juliadoc/jldoctest.py

```diff
@@ -480,14 +480,14 @@ def __run(self, test, out):
                 self.julia.stdin.write('_ans=ans; nothing\n'.encode('utf-8'))
                 # read separator
                 sep = 'fjsdiij3oi123j42'
-                self.julia.stdin.write(('println("' + sep + '")\n').encode('utf-8'))
+                self.julia.stdin.write(('println("\\n' + sep + '")\n').encode('utf-8'))
```

Comment:
does this become unnecessary if dump gets fixed / changed back? or is it harmless to have anyway?